Rakefile

```diff
@@ -17,8 +17,28 @@ namespace :test do
   task :all => [:unit, :integration]
 end
 
-YARD::Rake::YardocTask.new do |t|
-  t.files   = ['lib/features/**/*']
+namespace :docs do
+  YARD::Rake::YardocTask.new do |t|
+    t.files   = ['lib/features/**/*.rb']
+    t.options = ['--tag', 'step_input:Step parameters']
+  end
+
+  task :publish do
+    throw StandardError("Docs have not been generated") unless Dir.exist?("doc")
+    Dir.chdir("doc") do
+      `git init`
+      `git checkout -b gh-pages`
+      `git add .`
+      # `git commit -m "Docs update"`
+      # `git push -f git@github.com:bugsnag/maze-runner.git gh-pages`
```

Comment:
I'm being cautious on this command, as I don't want to accidentally overwrite non-"gh-pages" branches.  Can someone verify this for me?